src/test/kotlin/org/wfanet/measurement/eventdataprovider/noisers/GaussianNoiserTest.kt

```diff
@@ -56,11 +55,20 @@ class GaussianNoiserTest {
 
     val sigma = calculateStandardDeviation(samples)
     // Sigma value with pre-set epsilon and delta
-    val expectedSigma = 6.557822067460045
+    val expectedSigma = 6.55780908203125
     val diffRatio = abs((expectedSigma - sigma) / expectedSigma)
 
     assertThat(diffRatio).isLessThan(0.2)
   }
+
+  @Test
+  fun `solveSigma returns expected value`() {
+    val sigma = GaussianNoiser.getSigma(MEASUREMENT_SPEC.reachAndFrequency.reachPrivacyParams)
+    val expectedSigma = 6.55780908203125
+
+    assertThat(sigma).isEqualTo(expectedSigma)
```

Comment:
Wondering if `isEqualTo` is too strict. Would it possibly fail to due future version changes that affects the utils used in `GaussianNoiser.getSigma`, say, `standardNormalDistribution.cumulativeProbability`